### DISCUSSION TITLE
Claim an cyoa that 3rd party has uploaded

### DISCUSSION THREAD

**Discussion by pixelRealmCo** [DragonsWho/CYOACatalog#12] [Ideas]
A way to for authors to claim uploaded cyoas as their own if a 3rd party upload it so they can eddit info about it or even delete it.

**Comment by DragonsWho** [comment]
Yeah, I think I mentioned the confirmed authors personal page. 
After that they can edit game descriptions, but not sure about deleting them. I really don't like it when someone during a sudden impulse deletes content, even if they created it. It's especially stupid considering there's a hell of a lot of copies left online anyway.
